client/src/pages/EditForm.js

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import TextField from "@mui/material/TextField";
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const theme = createTheme();
+
+export default function EditForm() {
+  const [editing, setEditing] = useState(false);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    console.log({
+      email: data.get("email"),
+      password: data.get("password"),
+    });
+  };
+
+  // currently in progress
+  // function isEditing() {
+  //   console.log("edit mode activated");
+  //   setEditing();
+  // }
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Container component="main" maxWidth="xs">
+        <CssBaseline />
+        <Box
+          sx={{
+            marginTop: 8,
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+          }}
+        >
+          <Typography component="h1" variant="h5">
+            Edit
+          </Typography>
+
+          <Box
+            component="form"
+            onSubmit={handleSubmit}
```

Comment:
LOVE this!  registering the handler on submit vs on click is the best practice.  Many validation libraries even require it.

💪